tests/scripts/create-gke-cluster.sh

```diff
@@ -14,7 +14,8 @@ gcloud container clusters create ${GKE_CLUSTER_NAME} \
     --service-account ${GKE_NODES_SERVICE_ACCOUNT} \
     --enable-private-nodes \
     --master-ipv4-cidr 172.16.${ip_random_digit}.32/28 \
-    --metadata=block-project-ssh-keys=TRUE
+    --metadata=block-project-ssh-keys=TRUE \
+    --monitoring=SYSTEM,POD,DEPLOYMENT
```

Comment:
do we need to specify this command line flag to enable logging for workloads ?
https://cloud.google.com/sdk/gcloud/reference/container/clusters/create#--logging

Reply:
I didn't see the need for enabling logging at this time. Trying to minimize the amount of user intervention, and we can get logs directly from the pods using the k8s client.